src/main/java/com/kryptnostic/conductor/rpc/EntityDataLambdas.java

```diff
@@ -0,0 +1,26 @@
+package com.kryptnostic.conductor.rpc;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Function;
+
+public class EntityDataLambdas implements Function<ConductorSparkApi, Boolean>, Serializable {
```

Comment:
If you change to implements `Predicate<ConductorSparkApi>, Serializable` and make necessary changes in `KryoSerializer` you can avoid boxing and everything else should pretty much stay the same.